src/main/java/com/chessmates/league/LeagueTable.groovy

```diff
@@ -0,0 +1,70 @@
+package com.chessmates.league
+
+import com.chessmates.utility.GameColor
+import org.slf4j.Logger
+
+/**
+ * A simple league table.
+ */
+class LeagueTable {
+
+    private static final Logger logger = org.slf4j.LoggerFactory.getLogger(LeagueTable)
+
+    final winPoints
+    final loosePoints
+
+    private final Map<String, LeagueRow> playerRows = new HashMap()
+
+    LeagueTable(Integer winPoints, Integer loosePoints) {
```

Comment:
Happy to add - I wasn't too sure how draws were handled in chess, but more importantly I'm not sure how Lichess indicates a draw. Do you have an example of a drawn game? Perhaps a game you've drawn with someone?